src/frontend/index.ts

```diff
@@ -2,19 +2,38 @@ import sslify from 'express-sslify';
 import helmet from 'helmet';
 import getFrontendApp from './app';
 import {Application} from 'probot';
-import {Express, Router} from 'express';
+import {Express, NextFunction, Request, Response, Router} from 'express';
+import crypto from 'crypto';
 
 function secureHeaders(router: Router, frontendApp: Express) {
+
+  router.use((_: Request, res: Response, next: NextFunction): void => {
+    res.locals.nonce = crypto.randomBytes(16).toString('hex');
+    next();
+  });
+
+
   // Content Security Policy
   router.use(helmet.contentSecurityPolicy({
     directives: {
```

Comment:
Personally, I wouldn't fill out every directive option for Helmet since they have very sensible defaults:
```
default-src 'self';
base-uri 'self';
block-all-mixed-content;
font-src 'self' https: data:;
frame-ancestors 'self';
img-src 'self' data:;
object-src 'none';
script-src 'self';
script-src-attr 'none';
style-src 'self' https: 'unsafe-inline';
upgrade-insecure-requests
```
set `useDefaults: true` before the directives, then only change the directives that are specific to us. Docs here: https://github.com/helmetjs/helmet